pkg/kfapp/ossm/templates/control-plane/base/control-plane-ingress.patch.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: minimal
+  namespace: test
+spec:
+  proxy:
+    injection:
+      autoInject: true
+  gateways:
+    openshiftRoute:
+      enabled: false
+    ingress:
+      volumes:
+        - volume:
+            secret:
+              secretName: test-oauth2-tokens
+              optional: true
+          volumeMount:
+            name: test-oauth2-tokens
+            mountPath: "/etc/istio/test-oauth2-tokens"
+            readOnly: true
```

Comment:
Humm.... I think tests are overwriting the actual template now :D

Reply:
Fixed in 88224c7

Reply:
Ah okay, I was a little confused about why this was added, I should have asked.